recipes/crossdb/all/test_package/test_package.c

```diff
@@ -0,0 +1,10 @@
+#include <crossdb.h>
+
+int main () {
+	xdb_conn_t	*pConn = xdb_open (":memory:");
+	XDB_CHECK (NULL != pConn, printf ("failed to create DB\n"); return -1;);
+
+	xdb_close (pConn);
+
+    return 0;
```

Comment:
```suggestion
#include <stdio.h>
#include <stdlib.h>

#include <crossdb.h>

int main () {
    printf("Cross DB Version: %s\n", xdb_version());
    return EXIT_SUCCESS;
```

Simplify test package to avoid DB creation.